cmd/collector/app/span_processor.go

```diff
@@ -266,7 +268,7 @@ func (sp *spanProcessor) ProcessSpans(batch processor.Batch) ([]bool, error) {
 	return batchOks, batchErr
 }
 
-func (sp *spanProcessor) processSpans(batch processor.Batch, spans []*model.Span) ([]bool, error) {
+func (sp *spanProcessor) processSpans(_ context.Context, batch processor.Batch, spans []*model.Span) ([]bool, error) {
```

Comment:
just curious - why are we passing the context in here if its not being used?

Reply:
better have it and not need it